lib/libspl/assert.c

```diff
@@ -25,7 +25,13 @@
 
 #include <assert.h>
 
-int libspl_assert_ok = 0;
+static int libspl_assert_ok = 0;
```

Comment:
Purely out of curiosity: why?

Reply:
ASan reports ODR violation on `libspl_assert_ok` which leaks to ABI. +1 to changing to boolean.